src/utils.js

```diff
@@ -77,23 +77,6 @@ var utils = {
 		};
 	},
 
-	// @todo move this method in Chart.helpers.options.toFont
-	parseFont: function(value) {
```

Comment:
@kurkle this method has been [moved to the core lib](https://github.com/chartjs/Chart.js/pull/7383), which is great. However I realize that you also moved `color` under the font object (thus in this helper) which I think it's not great. It's not consistent with the way [CSS defines a font](https://developer.mozilla.org/en-US/docs/Web/CSS/font#Syntax). The reason why it triggers me some concerns is that I was planning to support `value` font string (e.g. `font: 'italic 12px "Fira Sans", serif`) as a shorthand for `font: { style: 'italic', size: 12px, family: "Fira Sans", serif }`. I really don't think having the color in this string makes sense and I would try to be consistent with CSS here.

Another reason to not have `color` under `font` is that the color isn't necessary only for the text. It can be used as the [base (fallback) color for borders, shadows, stroke, etc.](https://github.com/chartjs/chartjs-plugin-datalabels/blob/master/src/label.js#L232), if those properties are undefined (as defined in CSS).

Reply:
> Another reason to not have `color` under `font` is that the color isn't necessary only for the text. It can be used as the [base (fallback) color for borders, shadows, stroke, etc.](https://github.com/chartjs/chartjs-plugin-datalabels/blob/master/src/label.js#L232), if those properties are undefined (as defined in CSS).

In my poor opinion, the `color` for other elements is already present, Removing the color for font means that you can not specify a different color between font and elements. Instead, the `font color` could have the fallack to the `color`.

Even if the color is not part of CSS font representation, there are other items of the font which are not part of CSS if I'm not wrong `lineHeight`, for instance). 

I vote to maintain as is, because is self explaining.

Reply:
I failed to consider the CSS spec when doing that change. I agree we should try to be consistent with it.
`lineHeight` is actlually part of it, but `color`, `lineWidth` and `strokeStyle` are not. 

These can be debated, but I don't think we should. Lets let people around the CSS spec do that for us.

Disclaimer: I did not have a look how many collisions with `color`, but I think we mostly have `borderColor` and `backgroundColor`.

Reply:
> `lineHeight` is actlually part of it, but `color`, `lineWidth` and `strokeStyle` are not.

AFA I have seen, `lineHeight` is part of font object but not part of CSS font string, created by `helpers.canvas.toFontString`.

Reply:
https://developer.mozilla.org/en-US/docs/Web/CSS/font#Syntax

Reply:
Thank you @kurkle !! I was saying that `lineHeight` is not managed by `toFontString` method of `Chart.helpers.canvas` (and maybe it should be),  and not that's not part of web specification :D

Reply:
https://github.com/chartjs/Chart.js/pull/8066

Did not add the `lineHeight` in the font string, as our units are wrong (pixels instead of relative). That should probably be addressed too.